crowbar_framework/lib/crowbar/repository.rb

```diff
@@ -16,16 +16,46 @@
 
 module Crowbar
   class Repository
-    attr_reader :platform, :id, :registry
+    attr_reader :platform, :id, :config
 
     class << self
       def load!
-        @config = YAML.load_file(Rails.root.join("config/repos.yml"))
+        etc_yml = "/etc/crowbar/repos.yml"
+
+        @all_repos = YAML.load_file(Rails.root.join("config/repos.yml"))
+
+        if File.exists? etc_yml
```

Comment:
Lint/DeprecatedClassMethods: `File.exists?` is deprecated in favor of `File.exist?`.